app/mailers/user_notifier.rb

```diff
@@ -0,0 +1,12 @@
+class UserNotifier < ActionMailer::Base
+  default :from => 'any_from_address@example.com'
```

Comment:
To elaborate on Rails Config...

There is a config/settings.yml file that is a blank template.  In production, we deploy a config/settings.local.yml that overrides the values.  See [here](https://github.com/ManageIQ/miq_bot/blob/c524fbf51ccffb32a464bd0cdcb5a797cdf80eb9/app/models/commit_monitor_repo.rb#L49) for an example.